public/.well-known/security.txt

```diff
@@ -0,0 +1,25 @@
+-----BEGIN PGP SIGNED MESSAGE-----
+Hash: SHA512
+
+Contact: mailto:admins@quiltmc.org
+Expires: 2023-03-19T12:00:00.000Z
```

Comment:
Doesn't this key expire today?

Reply:
PSA: We're not in 2022
I'll fix it lmao